docs/content/users/install/ddev-installation.md

```diff
@@ -186,13 +186,15 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
     ```bash
     wsl --install
     # Reboot is often required after this command.
+    wsl --update
```

Comment:
Of course if somebody just did the install, the update won't do anything, but hopefully this will convince those who have it installed to do the update.